Src/Sarif.PartnerMatcher.Function/HttpAnalyzeFn.cs

```diff
@@ -0,0 +1,48 @@
+using System;
```

Comment:
for all .cs files, add the header that we use in the other files. If you check, it will probably show the intellisense.

Reply:
done